src/Persistence.LinnApps/Repositories/PurchaseOrderReqRepository.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Linn.Purchasing.Persistence.LinnApps.Repositories
+{
+    using System.Linq;
+
+    using Linn.Common.Persistence.EntityFramework;
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders;
+
+    using Microsoft.EntityFrameworkCore;
+
+    public class PurchaseOrderReqRepository : EntityFrameworkRepository<PurchaseOrderReq, int>
+    {
+        private readonly ServiceDbContext serviceDbContext;
+
+        public PurchaseOrderReqRepository(ServiceDbContext serviceDbContext)
+            : base(serviceDbContext.PurchaseOrderReqs)
+        {
+            this.serviceDbContext = serviceDbContext;
+        }
+
+        public override PurchaseOrderReq FindById(int key)
+        {
+            var purchaseOrderReq = this.serviceDbContext.PurchaseOrderReqs
```

Comment:
could just return this straight away

Reply:
Well spotted, left over from debugging so that I could see what the object it was getting from the db!